config/htcondor-ce.spec

```diff
@@ -76,6 +75,21 @@ Requires: /usr/bin/unshare
 %description
 %{summary}
 
+%package core
+Group: Applications/System
+Summary: Core configuration for HTCondor-CE
+
+Provides:  condor-ce = %{version}
+Provides:  %{name}-master = %{version}-%{release}
```

Comment:
I'm not sure if these `Provides:` should go into the core or the base package.

Reply:
Depends on what requires them. If whatever requires them will not work without the OSG bits too, then the Provides should go in the main package.

Reply:
@matyasselmeci ^^